packages/api/cli/src/electron-forge-import.ts

```diff
@@ -1,16 +1,15 @@
-import path from 'node:path';
-
 import { api } from '@electron-forge/core';
-import program from 'commander';
-import fs from 'fs-extra';
+import { program } from 'commander';
 
 import './util/terminate';
+import packageJSON from '../package.json';
```

Comment:
Note to future us, once we bump the minimum Node.js version in Forge, we should be able to use import attributes to change this to `import packageJSON from '../package.json' with { type: 'json' };`